### DISCUSSION TITLE
LEDs

### DISCUSSION THREAD

**Discussion by PedroTheIntern** [strayer/taira-keyboard#5] [General]
I'm looking at building this with the [MBK Glow keycaps](https://www.littlekeyboards.com/products/mbk-glow-keycaps).
What would be the easiest way for me to get some light behind the keycaps?

**Comment by strayer** [comment]
Hey,

you would have to modify the whole thing… meaning adding the relevant components to the KiCad schematic and then retrace the PCB. ZMK has support for backlight: https://zmk.dev/docs/features/backlight

I didn't bother with LEDs since the Taira is intended to run on battery all the time which doesn't really work with LEDs :) Its certainly possible but quite a bit of work.
